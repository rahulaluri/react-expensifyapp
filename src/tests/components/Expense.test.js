import ReactShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import {shallow} from 'enzyme';
import ExpenseListItem from '../../components/Expense';

const expenses = [{
    id:1,
    description:'love',
    note: '12312',
    amount: 12312,
    createdAt:0
}, {description:'loadave',
note: '123asd12',
amount: 0,
createdAt:0}];


test('Should render the expense item correctly', ()=>{
    const wrapper = shallow(<ExpenseListItem  {...expenses[0]}/>);
    expect(wrapper).toMatchSnapshot();
})
