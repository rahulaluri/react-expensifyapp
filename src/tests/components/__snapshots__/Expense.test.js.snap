// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the expense item correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      love
    </h3>
  </Link>
  <p>
    12312
    -
    0
  </p>
</div>
`;
